mls_validation_service/src/cached_signature_verifier.rs

```diff
@@ -70,18 +70,18 @@ mod tests {
         MultiSmartContractSignatureVerifier, SmartContractSignatureVerifier, ValidationResponse,
         VerifierError,
     };
-    use xmtp_id::utils::test::{smart_wallet, SignatureWithNonce, SmartWalletContext};
+    use xmtp_id::utils::test::{docker_smart_wallet, SignatureWithNonce, SmartWalletContext};
```

Comment:
These changes are due to test failures I ran into where it cannot find the `anvil` cli.